### ISSUE TITLE
Unusable midi IO in Ableton for some users

### ISSUE THREAD

**Issue by th8a** [wineasio/wineasio-old-docs#3]
Hello,

 I just wanted to drop in and point out a bug that has been effecting some people who are trying to use wineasio with Ableton live. It is quite strange in that ableton will otherwise work flawlessly and  will detect midi inputs and outputs. It will even show midi activity in the indicator when midi data is triggered, however, the midi data will never reach the midi track or instrument. If you fiddle with the midi settings while triggering midi, you can get a few notes to sound through a track, but then it simply goes dead again. The bug is discussed in great detail on this thread: https://forum.winehq.org/viewtopic.php?f=8&t=30766&p=119184#p119184 .  Hope this is helpful. Otherwise, Great work on wineasio. We music producers who are tired of being stuck with Apple or Windows have been waiting for the day when our favorite DAWs will run well in Linux. Aside from this bug, it's pretty much all there, so thank you and much appreciation. 

Cheers!

**Comment by mcnster** [comment]
You're welcome. :)

WineASIO does not and never has supported MIDI signals; it handles audio signals only, specifically ASIO audio signals.

But if you are still having trouble with Ableton Live, we can try to troubleshoot your problem.

**Comment by th8a** [comment]
Yes... I get that wineASIO is about audio and not MIDI... Interestingly, however, the problem described above is only present in Ableton when WineASIO is set as the audio driver. For instance, I tried to test it with the FL studio ASIO driver, midi worked flawlessly... but of course, then one can only use the pulse audio outputs in carla/patchage etc. Switch it back to wineASIO, MIDI starts buggin out... It's an odd bug indeed.  I posted this originally a while ago now... I should probably check with an updated version of ableton to see if it still persists, as I suspect the bug may possibly be on the Ableton side rather than the wineASIO side...
